### ISSUE TITLE
Requirements in README.md?

### ISSUE THREAD

**Issue by jrogers1126** [dnsviz/dnsviz#5]
Would be nice to have some requirements information in terms of python modules that need to be installed, and python version.

I'm fumbling through setting this up, and will eventually guess the right requirements (and may submit a proposed merge to README.md):

```
~/dev/dnsviz$ python2.7 ./setup.py 
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied

~/dev/dnsviz$ sudo python2.7 ./setup.py 
Traceback (most recent call last):
  File "./setup.py", line 54, in <module>
    'dnspython (==1.11)',
  File "/usr/lib/python2.7/distutils/core.py", line 112, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/lib/python2.7/distutils/dist.py", line 259, in __init__
    getattr(self.metadata, "set_" + key)(val)
  File "/usr/lib/python2.7/distutils/dist.py", line 1218, in set_requires
    import distutils.versionpredicate
  File "/usr/lib/python2.7/distutils/versionpredicate.py", line 5, in <module>
    import operator
ImportError: /usr/lib/python2.7/lib-dynload/operator.so: wrong ELF class: ELFCLASS32
```

**Comment by cdeccio** [comment]
What platform are you running on?  How was your python2.7 installed?

**Comment by jrogers1126** [comment]
RHEL5, I'll have to check with my sysadmin, but I believe he compiled from source (its in /usr/local/, so not installed via YUM)

**Comment by wfms** [comment]
So far the dependencies required to install this package are:

dnspython
M2Crypto
pygraphviz

**Comment by cdeccio** [comment]
Yes, and the minimum python version should be 2.7.

I'll add these (and build/install info) to the README.

**Comment by cdeccio** [comment]
Fixed by 2e7ee793f5dea5a052c691aaa2fa898c5fc1b364.
